From 635624b3cfa5f35e99fee95d7eeeded7df678c23 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Fri, 5 Jul 2024 06:53:36 +0200
Subject: [PATCH 1183/1695] plat-stm32mp1: calib: fix header files inclusion

Correct the order of the header files inclusion that should follow
the alphabetical order as per OP-TEE OS coding style and remove
inclusion of kernel/timer.h that is present twice.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4ac92a75dfa04e3e4567ba470df828a5f17cb247 plat-stm32mp1: calib: fix header files inclusion]
Change-Id: Ia78b084a3dd7af1cee685a5af085e4db93f3eba1
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/392000
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/drivers/stm32mp1_calib.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_calib.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_calib.c
index f73147603..a549cd18e 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_calib.c
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_calib.c
@@ -16,17 +16,16 @@
 #endif
 #include <initcall.h>
 #include <io.h>
-#include <libfdt.h>
 #include <keep.h>
 #include <kernel/boot.h>
 #include <kernel/delay.h>
 #include <kernel/dt.h>
 #include <kernel/interrupt.h>
-#include <kernel/timer.h>
 #include <kernel/panic.h>
 #include <kernel/pm.h>
 #include <kernel/pseudo_ta.h>
 #include <kernel/timer.h>
+#include <libfdt.h>
 #include <limits.h>
 #include <mm/core_memprot.h>
 #include <stm32_pta_calib.h>
-- 
2.47.2

